libindy/src/commands/metrics.rs

```diff
@@ -0,0 +1,43 @@
+use std::cell::RefCell;
+use std::collections::HashMap;
+use std::rc::Rc;
+
+use crate::domain::ledger::request::ProtocolVersion;
+use crate::domain::pool::{PoolConfig, PoolOpenConfig};
+use indy_api_types::errors::prelude::*;
+use indy_api_types::CommandHandle;
+
+pub enum MetricsCommand {
+    CollectMetrics(Box<dyn Fn(IndyResult<()>) + Send>)
+}
+
+pub struct MetricsCommandExecutor {
+    close_callbacks: RefCell<HashMap<CommandHandle, Box<dyn Fn(IndyResult<()>)>>>,
+    refresh_callbacks: RefCell<HashMap<CommandHandle, Box<dyn Fn(IndyResult<()>)>>>,
+    // TODO: change  PoolHandle to MetricsHandler
+    open_callbacks: RefCell<HashMap<CommandHandle, Box<dyn Fn(IndyResult<PoolHandle>)>>>,
```

Comment:
Do we really use/need these callbacks, or they are also leftovers from wallet command executor?